packages/core/src/utilities.ts

```diff
@@ -63,3 +63,16 @@ export function applyStyles(
 
   return flattenedStyles;
 }
+
+export function getRealValue(value: any) {
+  // console.log("getRealValue typeof", typeof value);
+  // console.log("getRealValue value", value);
+  switch (typeof value) {
```

Comment:
i would personally use lodash' `isString` or `isNumber` which does some fancier stuff under the hood to absolutely make sure:

you can install:
lodash.isstring
lodash.isnumber

we use these in a couple of places

and then 
```
if (!isString(x) && !Number(x)) { throw ... }
```

Reply:
@peterpme done.

Also, you might want to consider this lib over lodash for the future: https://anguscroll.com/just/

Reply:
@peter. Done.